### DISCUSSION TITLE
Generac 24kw

### DISCUSSION THREAD

**Discussion by rwskinner** [jgyates/genmon#1131] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Version

Latest

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have not submitted logs

### What is/are your question(s)?

Why does the amp meter redline at 50 amps when the generac 24kw is good for 100 amps?
I was pulling 14kw out of 24kw and my amp meter was in the red.  Just asking.

**Comment by jgyates** [comment]
I may be able to answer this, I might need more info to answer your questions. 

1) I assume this is a air cooled model, let me know if that is not correct
2) The amp gauge for the air cooled shows leg one and leg two current. For a 24kw model each each leg should have a max of 50A with 2 legs equaling 100A. Is this the gauge you are referring? If not, specifically are you referring too?
3) Is genmon identifying your controller and rated kW correctly. Logs would confirm that. 

The reason the max current is divided between 2 legs is due to the design of the generator. Each leg has an unbalanced capacity of 50% of the total capacity.

**Reply by skipfire** [reply]
@rwskinner under settings what does GenMon show for `Maximum kW Output`?  Also where does your power output gauge top out?  I have a 22kW and the gauge goes red at 20 and stops a bit past the 25 mark.

24kW should be 100A per leg (120v * 100A = 12kW * 2 legs = 24kW), but my 22kW shows correctly in GenMon so I'm guessing this is a detection or configuration issue.
![image](https://github.com/jgyates/genmon/assets/10307327/ed184978-573d-4997-af0b-d7fd475006ce)

**Reply by rwskinner** [reply]
@jgyates  Yes, Air Cooled.  I submitted logs a few moments ago.
It's rated for 100 amps max on propane.  The generator has a 100 amp breaker installed in the chassis, which protects at 100 amps per leg.

@skipfire It's set to 24kw.  
![image](https://github.com/jgyates/genmon/assets/10195178/0a6def0a-4d14-42d9-9d93-ad0da8a83772)
![image](https://github.com/jgyates/genmon/assets/10195178/e54d3a06-b8bf-4095-9a0f-0883ffca0a3f)

**Reply by rwskinner** [reply]
I did not change a thing except for upgrading from  1.19.02 to .03 and afterwards the gauge changed.

![image](https://github.com/jgyates/genmon/assets/10195178/98b33c84-88ca-4f86-ac32-24cd7bb5ceee)

**Reply by jgyates** [reply, marked as answer]
@rwskinner 

They way it is displaying is correct for your generator. The addition of L1 and L2 gauges and the load imbalance detection were added at different days so I it is possible that the version you had previously was displaying incorrectly. I recall making a change that corrected the L1, L2 max current between 1.19.2 and 1.19.3.

**Reply by rwskinner** [reply]
All seems fine now.  Thanks for the support and such an awesome app!
